src/expression.rs

```diff
@@ -172,6 +174,12 @@ fn calculate_function(
     }
 }
 
+/// Is this a small floating point number?
+#[inline(always)]
+fn _is_small(x: f64) -> bool {
```

Comment:
I think I'd `s/_is_small/is_small`, as it's already private unless marked `pub` otherwise. But maybe you're trying to indicate something else?

Reply:
Great point! I've been writing a bunch of Python recently. I'll update `_hash_to_u64` as well (since I'm the guilty party there as well).